src/components/Column/Column.scss

```diff
@@ -172,7 +180,7 @@
   min-width: $icon--medium;
   min-height: $icon--medium;
   margin: 0 $margin--small 0 0;
-  color: $color-middle-gray;
+  color: $color-warning-red;
```

Comment:
![image](https://user-images.githubusercontent.com/44020029/206432022-09ff8bfa-cfd7-4f55-b59c-b2f1b4574de2.png)

What do you think about a border around the icon?

Reply:
```suggestion
  color: $color-warning-red;
  border: 2px solid $color-warning-red;
  box-sizing: content-box;
  border-radius: $border-radius--round;
```

Reply:
Oh and in case you want to use this ^, you should probably add the border-color to the hover effect, too :)

Reply:
Ah, I think it look's better and more minimalistic without the border 😄